core_lib/interface/timecontrols.cpp

```diff
@@ -70,21 +70,17 @@ TimeControls::TimeControls(TimeLine *parent ) : QToolBar( parent )
     mJumpToEndButton= new QPushButton();
     mJumpToStartButton= new QPushButton();
 
-    QLabel* separator = new QLabel();
-    separator->setPixmap(QPixmap(":icons/controls/separator.png"));
-    separator->setFixedSize(QSize(37,31));
-
     QIcon loopIcon(":icons/controls/loop.png");
     QIcon soundIcon(":icons/controls/sound.png");
-    QIcon endplayIcon(":icons/controls/endplay.png");
-    QIcon startplayIcon(":icons/controls/startplay.png");
+    QIcon JumpToEndButton(":icons/controls/endplay.png");
+    QIcon JumpToStartButton(":icons/controls/startplay.png");
```

Comment:
Can you rename these variables to start with a lowercase letters to keep the naming styles consistent?